samples/buildpacks/stacks/bionic/base/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:bionic
```

Comment:
WDYT about using cloudfoundry/cnb-base, cnb-run and cnb-build instead so we don't re-invent the wheel?

Reply:
The main problem I ran into while trying that is, the image changes the user. This prevents a container built from that image to be able to install anything else.

Reply:
That being said, I agree with you and have filed an issue to see if the idea of appending to their images would be possible: https://github.com/buildpack/stacks/issues/6

Reply:
SGTM